CHANGELOG.md

```diff
@@ -2,12 +2,56 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
-## Unreleased
+## v0.2.0.0 Berlin
+We are happy to release solang `v0.2.0` codename `Berlin` today. Aside from containing many smaller fixes and improvements, this release marks a milestone towards maturing our Substrate compilation target: Any regressions building up since `ink!` v3.0 are fixed, most notably the metadata format (shoutout and many thanks to external contributor [extraymond](https://github.com/extraymond)) and event topics. Furthermore, we are leaving `ink!` version 3 behind us, in favor of introducing compatibility with the recent `ink!` 4 beta release and the latest pallet contract `v0.22.1`.
 
 ### Added
-- Introduce new sub-command `solang idl` which can be used for generating
+- **Solana / breaking:** The try-catch construct is no longer permitted on Solana, as it
+   never worked. Any CPI error will abort the transaction.
+   [seanyoung](https://github.com/seanyoung)
+- **Solana:** Introduce new sub-command `solang idl` which can be used for generating
   a Solidity interface file from an Anchor IDL file. This can be used for calling
   Anchor Contracts on Solana. [seanyoung](https://github.com/seanyoung)
+- **Substrate:** Provide specific Substrate builtins via a "substrate" file. The
+  `Hash` type from `ink!` is the first type made available for Solidity contracts.
```

Comment:
`is the first type made available for Solidity contracts` 
This needs rephrasing. Is it the first Substrate specific type?

Reply:
Exactly